packages/cozy-codemods/src/transforms/transform-radio.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Original:
+ *  <Radio
+ *    value="radioValue1"
+ *    label="This is a radio button"
+ *    error={true}
+ *    disabled={false}
+ *    gutter={true}
+ *  />
+ * New :
+ *  <FormControlLabel
+ *    value="radioValue1"
+ *    label="This is a radio button"
+ *    control={<Radio disabled={false} color="secondary" edge="start" />}
+ *  />
+ */
```

Comment:
maybe, this can go in a test, to check the behaviour of this utils :-)

Reply:
We should definitely add a test for it. We already have some tools to do so. Check https://github.com/cozy/cozy-libs/blob/master/packages/cozy-codemods/src/transforms/__tests__/modal-to-dialog.spec.js

Reply:
test ajouté